### DISCUSSION TITLE
New on GitHub/ IT and Translation

### DISCUSSION THREAD

**Discussion by marcusmangetti** [community/community#123406] [New to GitHub]
Hi, everyone!
I'm Marcus and one of my main interests here is discussion on translation API services privacy and use of data (Google and MS). Both claims not to use our data to train and improve their services, but translators' community is not particularly convinced to trust these terms of use. I'd like to connect to fellow IT e Translators already working on these issues. (EN and PT-BR)

**Comment by mandeep73k** [comment]
How can I upgrade my account from personal to student pack I don't have edu mail but I have ID card provided by University
